src/selectivity-base.js

```diff
@@ -27,7 +27,6 @@ var EventDelegator = require('./event-delegator');
  *         executed on the first element in the set of matched elements.
  */
 function selectivity(methodName, options) {
-    /* jshint validthis: true */
```

Comment:
Usage of this is not enforced for now, but it can be enabled with: no-invalid-this: 2, this rule will detect invalid usage of this in four files: 
gulp/tasks/browserify.js
src/selectivity-base.js
src/selectivity-dropdown.js
src/selectivity-traditional.js